configure.ac

```diff
@@ -56,6 +56,10 @@ AC_ARG_WITH([mpi],
              LDFLAGS="-L$withval/$mpi_libdir $LDFLAGS"],
             [])
 
+AC_ARG_ENABLE([tests], [AS_HELP_STRING([--disable-tests],
+      [Disable build of test binaries])], [enable_tests], [])
```

Comment:
nit: Do we need `enable_tests` here as the third argument? Can we simply use `x$enable_tests` directly on L61?